test/unit/test-purifier.js

```diff
@@ -19,14 +19,17 @@ import {
   purifyTagsForTripleMustache,
 } from '../../src/purifier';
 
+const html = document.createElement('html');
+const documentEl = {documentElement: html};
```

Comment:
Test setup code should be in a `beforeEach()`.

Reply:
done